templates/zerver/help/format-your-message-using-markdown.md

```diff
@@ -183,6 +184,16 @@ Silent mention: @_**Polonius** (@_ instead of @)
 
 ![](/static/images/help/markdown-status.png)
 
+## Timestamps
+
+Learn more about timestamps [here](/help/send-timestamps).
+
+```
+Let's have the next meeting on !time(2020-05-28T13:30:00+05:30)
```

Comment:
How about "Our next meeting is scheduled for "!time(...)"?  We'll need to remake the screenshot for it.